packages/phrases/src/locales/en.ts

```diff
@@ -119,6 +119,15 @@ const translation = {
       token_endpoint: 'Token Endpoint',
       user_info_endpoint: 'User Info Endpoint',
       save_changes: 'Save Changes',
+      more_options: 'More Options',
+      options_delete: 'Delete',
+      reminder: 'Reminder',
+      delete_description:
+        'This action cannot be undone. This will permanently delete the this application. Please enter the application name <span>{{name}}</span> to proceed.',
```

Comment:
why is the `<span>` required here? Is it necessary?

Reply:
Look at the comment above, I use this to locate the string which is needed to be highlighted.

Reply:
Got it, thx